README.md

```diff
@@ -1,3 +1,25 @@
 ![](assets/party.gif)
 # Eggy-Bowl
 This is a Egg catcher Game. I designed and developed it. I do represent it with name "Eggy-Bowl". It is a 3-D graphical game using Multiple python libraries.
+
+This Game can be played using arrow keys.
+
+Left arrow key : to move catcher left
+
+Right arrow key : to move catcher right
+
+Built With
+### Python
+### tkinter
+### itertools
+### random
+
+Getting Started
+### To play download .exe and play.
+
+### To create your own download python and getstarted.
```

Comment:
If you follow my advice, this would be:
### Getting started
- To play download .exe and play.
- To create your own download python and get started